modules/navier_stokes/test/tests/finite_volume/ins/turbulence/lid-driven/tests

```diff
@@ -319,6 +319,25 @@
       custom_cmp = 'lid-driven-low-turb.cmp'
     []
   []
+  [lid_driven_turb_buoyant]
+    requirement = 'The system shall be able to run fluid flow with k-epsilon turbulence model in enclosures with buoyant production, '
+    [result]
+      type = 'Exodiff'
+      input = 'lid-driven-turb-energy-wall-bouyancy-production.i'
+      exodiff = 'lid-driven-turb-energy-wall-bouyancy-production_out.e'
+      detail = 'and reach converged results with segregated solvers.'
+      abs_zero = 1e-6
+      heavy = true
+    []
```

Comment:
I think it is enough to have the run part here as a regression test. The real runs will come to open pronghorn soon.

Reply:
Thanks. Removed the results test